tests/asset-store/pkg/mockice/mockice.go

```diff
@@ -0,0 +1,374 @@
+package mockice
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/dynamic"
+)
+
+var (
+	podPort      int32  = 8080
+	svcPort      int32  = 80
+	defaultImage string = "hudymi/mockice:0.1.3"
```

Comment:
Note to self: add proper env so that we use image based on env var, and not default